acceptance/analyzer_test.go

```diff
@@ -876,32 +852,11 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 				})
 
 				when("auth registry", func() {
-					var authRegAppImage, authRegAppOtherImage, appAuthConfig string
-
-					it.Before(func() {
-						metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "app_image_metadata.json"), platform.LayersMetadata{})
-						authRegAppImage, appAuthConfig = buildAuthRegistryImage(
-							t,
-							"some-app-image-"+h.RandString(10),
-							filepath.Join("testdata", "analyzer", "app-image"),
-							"--build-arg", "fromImage="+containerBaseImage,
-							"--build-arg", "metadata="+metadata,
-						)
-
-						authRegAppOtherImage, appAuthConfig = buildAuthRegistryImage(
-							t,
-							"some-app-image-"+h.RandString(10),
```

Comment:
As we can see here, the app name was called `some-app-image` and now we changed it to `some-other-app-image`.
I think there is no problem in this change, but just wanted to double check with you that you're aware of it.